Framework/Kernel/test/TimeROITest.h

```diff
@@ -145,6 +145,20 @@ class TimeROITest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(value.durationInSeconds() / ONE_DAY_DURATION, 0.);
   }
 
+  void test_Mask() {
```

Comment:
There may be a problem with `addROI()`. I did:
```cpp
  void test_addROI() {
    TimeROI value0{TWO, THREE};
    value0.addROI(ONE, TWO);
    value0.addROI(THREE, FOUR);
    std::string out = value0.debugStrPrint();
```
and got 
```
0: 2023-Jan-01 00:01:00 to 2023-Jan-03 00:01:00
```
I was expecting `Jan-04` as stop time. Can you please verify?

Reply:
Also, executing the last line here throws and exception:
```
    TimeROI value0{ONE, TWO};
    value0.addROI(THREE, FOUR);
    value0.addROI(TWO, THREE);
```
the message:
```
********* UNHANDLED EXCEPTION *********
  what(): In TimeROI::addROI: Values are not unique
```
The expected result should be
```
0: 2023-Jan-01 00:01:00 to 2023-Jan-04 00:01:00
```